pkgs/development/tools/misc/netcoredbg/default.nix

```diff
@@ -49,19 +47,32 @@ let
     inherit pname version src dotnet-sdk;
 
     projectFile = "src/managed/ManagedPart.csproj";
+    # nix run .#legacyPackages.x86_64-linux.netcoredbg.passthru.fetch-deps
```

Comment:
Ideally this should be put into the update script. dotnet has a generic one which should be extended.

Reply:
added a simple updateScript, tested and seems to work ok